### DISCUSSION TITLE
How to add image in another image keeping original color ?

### DISCUSSION THREAD

**Discussion by schlebe** [ImageMagick/ImageMagick#5847] [Help] [answered]
I search to add an PNG image into another PNG image to compose an A4 paper.

I use following PowerShell code

```
$Magick = "D:\Program Files\ImageMagick-7.1.0-Q16-HDRI\magick.exe"
& $Magick liste-des-chaines.png i.png -geometry +20+20 -composite liste-des-chaines.png

```
When I look the result, I can see that image have been added ... but inserted image has lost his color ! All color are replaced by black/white (or grey)  image.
![image](https://user-images.githubusercontent.com/16349543/205495870-93f45f25-ffb2-463a-a063-522dea8535e6.png)

What is happening ? What can I do to keep colors ?

For information, inserted image before insertion is following

![image](https://user-images.githubusercontent.com/16349543/205495988-ababb52a-8fb5-42b7-a92e-e81f10d89cee.png)

Thanks for any help.

**Comment by snibgo** [comment, marked as answer]
Your `liste-des-chaines.png` is grayscale, so that metadata is carried to the output. Insert "`-set colorspace sRGB`" after reading that file to give it colour channels.

**Comment by schlebe** [comment]
Thanks, I have modified my command as proposed

`    & $Magick liste-des-chaines.png i.png -colorspace sRGB -geometry +$X+$Y -composite liste-des-chaines.png`

and now ... that is working correctly.
